templates/course_template_list.mustache

```diff
@@ -106,4 +106,4 @@
             </div>
         {{/groups}}
     {{/templates}}
-</div>
\ No newline at end of file
+</div>
```

Comment:
I don't know what this change is about. I just edited line 81 directly on the GitHub website.